java-tests/runtime/src/test/kotlin/io/spine/validate/option/RequiredChoiceSpec.kt

```diff
@@ -50,12 +53,14 @@ internal class IsRequiredSpec : ValidationOfConstraintTest() {
             }
         }
         val violation = exception.constraintViolations[0]
+        val oneof = Meal.getDescriptor().oneofs[0]
+        violation.fieldPath.joined shouldBe oneof.name
```

Comment:
Here it is seen that I have put path to `oneof` into `fieldPath: FieldPath` of `ConstraintViolation`. 

As a simple solution, we can treat `oneof` as a field within our public and internal API. The difference is insignificant from a user point of view.